From 712fab729d0ae178efa11bc359a8045402c98150 Mon Sep 17 00:00:00 2001
From: hoahuynh <hoa.huynh.ud@renesas.com>
Date: Mon, 18 Nov 2019 11:05:01 +0700
Subject: [PATCH] flash-writer: makefile: Modify Makefile

This patch modifies 2 points as below:
- Typo OBJDMP: must be OBJDUMP
- Use environment toolchain if defined.

Signed-off-by: hoahuynh <hoa.huynh.ud@renesas.com>
---
 makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/makefile b/makefile
index c6d5dec..84d9d08 100644
--- a/makefile
+++ b/makefile
@@ -179,17 +179,17 @@ DEPEND_FILE = $(patsubst %.lib, ,$(OBJ_FILE:%.o=%.d))
 
 ###################################################
 #C compiler
-CC = $(CROSS_COMPILE)gcc
+CC ?= $(CROSS_COMPILE)gcc
 #Assembler
-AS = $(CROSS_COMPILE)as
+AS ?= $(CROSS_COMPILE)as
 #Linker
-LD = $(CROSS_COMPILE)ld
+LD ?= $(CROSS_COMPILE)ld
 #Liblary
-AR = $(CROSS_COMPILE)ar
+AR ?= $(CROSS_COMPILE)ar
 #Object dump
-OBJDMP = $(CROSS_COMPILE)objdump
+OBJDUMP ?= $(CROSS_COMPILE)objdump
 #Object copy
-OBJCOPY = $(CROSS_COMPILE)objcopy
+OBJCOPY ?= $(CROSS_COMPILE)objcopy
 
 #clean
 CL = rm -rf
-- 
1.9.1

